FCCee/IDEA/compact/IDEA_o1_v03/VertexComplete_IDEA_o1_v03.xml

```diff
@@ -177,24 +179,27 @@
         <!-- Vertex outer barrel parameters -->
             <constant name="VTXOB_Staves1" value="23"/>
             <constant name="VTXOB_Staves2" value="51"/>
-            <constant name="VTXOB_nModules1" value="8"/>            <!-- 163.1 mm-->
-            <constant name="VTXOB_nModules2" value="16"/>           <!-- 326.3 mm -->
+            <constant name="VTXOB_nModules1" value="8"/>            <!-- 183.5 mm-->
+            <constant name="VTXOB_nModules2" value="16"/>           <!-- 346.7 mm -->
             <constant name="VTXOB_mod_z_spacing" value="0.2*mm"/>   <!-- Must be done for disks as well! Currently not done -->
     
             <constant name="VTXOB_z1" value="(VTXOB_nModules1*VTXO_mod_z+(VTXOB_nModules1-1)*VTXOB_mod_z_spacing)/2.0"/>
             <constant name="VTXOB_z2" value="(VTXOB_nModules2*VTXO_mod_z+(VTXOB_nModules2-1)*VTXOB_mod_z_spacing)/2.0"/>
-            <constant name="VTXOB_r1" value="VTXOB_r_min_layer"/>   <!-- 140 mm-->
+            <constant name="VTXOB_r1" value="VTXOB_r_min_layer"/>   <!-- 130 mm-->
             <constant name="VTXOB_r2" value="VTXOB_r_max_layer"/>   <!-- 315 mm -->
 
             <constant name="VTXOB_support_thickness"                value="2.0*VTXO_support_Fleece_thickness+VTXO_support_CarbonFiber_thickness+VTXO_support_PaperGraphite_thickness"/>
             <constant name="VTXOB_Pipe_offset"                      value="21.18*mm/2.0"/>
             <constant name="VTXOB_TrussStructure_thickness1"        value="0.501*mm"/> <!-- 326 mm length * 8.4 mm width / 5464.3 mm^3 volume = 0.501 mm of proxy carbon fiber thickness for light-weight truss structure -->
             <constant name="VTXOB_TrussStructure_thickness2"        value="0.477*mm"/> <!-- 652 mm length * 8.4 mm width / 11473.8 mm^3 volume = 0.477 mm of proxy carbon fiber thickness for light-weight truss structure -->
-            <constant name="VTXOB_TrussStructure_distance"          value="3.3*mm"/>
+            <constant name="VTXOB_TrussStructure_distance"          value="4.0*mm"/>
     
             <constant name="VTXOB_offset1" value="10.0*mm"/>
             <constant name="VTXOB_offset2" value="20.0*mm"/>
 
+            <constant name="VTXOB_zoffset1" value="-0.5*mm"/> <!-- To make sure no gap at theta=90 degree exists-->
+            <constant name="VTXOB_zoffset2" value="+0.5*mm"/> <!-- To make sure no gap at theta=90 degree exists-->
```

Comment:
Hi @armin-ilg, for my understanding - will this new `zoffset` be automatically captured by the segmentation? (so that the global position of the same `cellID` is now shifted by +-0.5mm)

Reply:
Hi. Yes, this offset is applied to all corresponding volume and surface placements.

Reply:
@armin-ilg Thanks!